src/Mvc/test/Mvc.FunctionalTests/SystemTextJsonOutputFormatterTest.cs

```diff
@@ -29,67 +29,13 @@ public override async Task Formatting_StringValueWithUnicodeContent()
             Assert.Equal("\"Hello Mr. \\ud83e\\udd8a\"", await response.Content.ReadAsStringAsync());
         }
 
-        [Fact]
-        public override async Task Formatting_SimpleModel()
-        {
-            // Arrange
-            var expected = "{\"Id\":10,\"Name\":\"Test\",\"StreetName\":\"Some street\"}";
-
-            // Act
-            var response = await Client.GetAsync($"/JsonOutputFormatter/{nameof(JsonOutputFormatterController.SimpleModelResult)}");
-
-            // Assert
-            await response.AssertStatusCodeAsync(HttpStatusCode.OK);
-            Assert.Equal(expected, await response.Content.ReadAsStringAsync());
-        }
-
-        [Fact]
-        public override async Task Formatting_CollectionType()
```

Comment:
Why remove these tests?

Reply:
They were overriding the baseline tests https://github.com/aspnet/AspNetCore/blob/master/src/Mvc/test/Mvc.FunctionalTests/JsonOutputFormatterTestBase.cs#L117-L129 because they were different from what we would want (i.e did not do camel-case). Now that it matches the baseline, we can have it call the one on the base.